x-pack/plugins/event_log/generated/mappings.json

```diff
@@ -267,6 +267,42 @@
                         }
                     }
                 },
+                "alert": {
+                    "properties": {
+                        "rule": {
+                            "properties": {
+                                "execution": {
+                                    "properties": {
+                                        "uuid": {
+                                            "type": "keyword",
+                                            "ignore_above": 1024
+                                        },
+                                        "status": {
+                                            "type": "keyword",
+                                            "ignore_above": 1024
+                                        },
+                                        "status_order": {
+                                            "type": "integer"
+                                        },
+                                        "metrics": {
+                                            "properties": {
+                                                "indexing_duration_sum_ms": {
```

Comment:
Agree with this. I would prefer to use a single unit for all durations in this schema and we already store things in nanoseconds per ECS standards.

Reply:
For our use case, nanoseconds are too precise. For example, some durations, like the execution gap duration, can potentially be days, and storing that kind of information in nanoseconds doesn't make much sense. On top of that, storing data in nanoseconds requires additional conversion logic to be implemented on reading and writing, which is not always convenient. That's why we decided to write durations in custom formats and add suffixes for more clarity.

> we already store things in nanoseconds per ECS standards.

@ymao1 I tried to find the ECS standard regarding storing time durations in nanoseconds but haven't seen anything. Is it in the docs or somewhere else? AFAIK, ECS doesn't impose any restrictions on custom fields, but maybe I'm missing something.

Reply:
Sorry, you are correct that there is no standard for custom duration fields. What I meant was since there is an `event.duration` field in the ECS standard that is specified in nanoseconds, it might be clearer if all other durations were in the same unit, although I agree, nanoseconds is a bit unwieldy.